com.unity.render-pipelines.core/Documentation~/View-Lighting-Tool.md

```diff
@@ -0,0 +1,23 @@
+# View Lighting Tool
+
+View Lighting Tool is a tool that allow to setup lighting in the camera space. The main Camera (Game View) is used to setup the orientation of a light with spherical coordinate. We can choose a target and Yaw, Pitch and Roll (useful when a cookie or IES is setuped).
+
+![](Images/ViewLightingTool00.png)
+
+## Using View Lighting Tool
```

Comment:
Wrote a gdoc for this instead of doing it line by line: https://docs.google.com/document/d/1P95QWtDSMva8H61BgWYq_4B0eb4rfTS4ySB7f6k5QcI/edit?usp=sharing

We shouldn't be using so many gifs and just document the steps instead! Can you take a look at this draft and let me know if its okay?